HISTORY.md

```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a
 Changelog](https://keepachangelog.com/en/1.0.0/), and this project
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.1.6](https://github.com/uploadcare/pyuploadcare/compare/v4.1.5...v4.1.6) - soon
```

Comment:
I think such updates should be minor, not micro, because they add features.

Reply:
It'll be 4.2.0